apps/condo/domains/ticket/utils/serverSchema/propertyPercentDataMapper.js

```diff
@@ -0,0 +1,25 @@
+const sum = require('lodash/sum')
+
+const getFieldPercentValue = (row, constants) => {
+    const { totalCounts, address } = constants
+    const statusCounts = Object.entries(row).filter(obj => obj[0] === address).map(e => e[1])
+    return (sum(statusCounts) / totalCounts[address] * 100).toFixed(2) + ' %'
+}
+
+const propertyPercentDataMapper = ({ row = {}, constants = {} }) => ({
+    header: {
+        ru: ['Адрес', 'В работе', 'Выполнена', 'Отменена', 'Отложена', 'Закрыта', 'Открыта'],
```

Comment:
guess we need to prefetch this data from real TicketStatuses

Reply:
We cannot do this since at the moment the template used for generation is not created at the time of the request. This is due to the issue of preserving custom links and sorts.